geth/params/defaults.go

```diff
@@ -12,6 +12,9 @@ const (
 	// StatusDatabase path relative to DataDir.
 	StatusDatabase = "status-db"
 
+	// DeduplicatorDatabase path relative to DataDir.
+	DeduplicatorDatabase = "status-deduplicator-db"
```

Comment:
Does it has to be an own database? Maybe Status DB with prefix?

Reply:
Initially, I thought about having different DBs per account. That was a preparation.
Since I ended up scrapping this idea, I'll fix that.